src/components/Contacts/ContactDetails/ContactDetailsHeader/DeleteContactModal/DeleteContactModal.tsx

```diff
@@ -187,6 +188,21 @@ export const DeleteContactModal: React.FC<DeleteContactModalProps> = ({
                 </ListItem>
               )}
             </List>
+            <br />
+            <Typography>
+              <Link
+                href={`mailto:${
+                  process.env.DONATION_SERVICES_EMAIL
+                }?subject=Request+contact+deletion&body=${encodeURIComponent(
+                  `Dear Donation Services,\n\Please could you remove the following contact: ${contactName} ` +
```

Comment:
I would imagine we would at least want the contact's partner number so that Donation Services knows which partner the user is referring to.